### PR TITLE
Shelter: need to add third party depends on in README

### PR DESCRIPTION
**Pull request by zhiminghufighting**
Fixes: #1104
Signed-off-by: zhiminghufighting <zhiming.hu@intel.com>

### CODE REVIEW

File: shelter/README.md [hunk 1]
```diff
@@ -102,3 +102,17 @@ shelter mrverify
...
+
+| Name | Repo URL | Licenses |
+| :--: | :-------:   | :-------: |
+| restruct | https://github.com/go-restruct/restruct                    | ISC |
+| cli | https://github.com/urfave/cli                    | MIT |
+| md2man | https://github.com/cpuguy83/go-md2man/v2/md2man                    | MIT |
+| sanitized_anchor_name | https://github.com/shurcooL/sanitized_anchor_name                    | MIT |
+| errors | https://github.com/pkg/errors                    | BSD-2-Clause |
+| v2 | https://github.com/russross/blackfriday/v2                    | BSD-2-Clause |
+| shelter | https://github.com/inclavare-containers/shelter                    | Apache-2.0 |
```
Comment (haosanzi) [inline shelter/README.md:118]: Does shelter depend on Shelter?

Comment (zhiminghufighting) [reply shelter/README.md:118]: You can see here:



import (	
"fmt"	
"github.com/inclavare-containers/shelter/remoteattestation"	
"github.com/inclavare-containers/shelter/utils"	
"github.com/urfave/cli"	
"strings")

Comment (jiazhang0) [reply shelter/README.md:118]: But it doesn't require to do it.

File: shelter/README.md [hunk 2]
```diff
@@ -102,3 +102,16 @@ shelter mrverify
...
+Direct Dependencies
+
+| Name | Repo URL | Licenses |
+| :--: | :-------:   | :-------: |
+| restruct | https://github.com/go-restruct/restruct                    | ISC |
+| cli | https://github.com/urfave/cli                    | MIT |
+| md2man | https://github.com/cpuguy83/go-md2man/v2/md2man                    | MIT |
+| sanitized_anchor_name | https://github.com/shurcooL/sanitized_anchor_name                    | MIT |
+| errors | https://github.com/pkg/errors                    | BSD-2-Clause |
+| v2 | https://github.com/russross/blackfriday/v2                    | BSD-2-Clause |
```
Comment (jiazhang0) [inline shelter/README.md:117]: Not v2. The package should be blackfriday.

Comment (zhiminghufighting) [reply shelter/README.md:117]: yes, thx!

File: shelter/README.md (hunk 1 above)
Comment (zhiminghufighting) [reply shelter/README.md]: remove shelter itself dependence.

**Pull request merged**